PhilightPtyLtd.Sharperlight version 5.4.60

DeusMaximus opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90716)

DeusMaximus committed:
PhilightPtyLtd.Sharperlight version 5.4.60

stephengillie commented on manifests/p/PhilightPtyLtd/Sharperlight/5.4.60/PhilightPtyLtd.Sharperlight.installer.yaml:
```diff
@@ -0,0 +1,21 @@
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://resources.sharperlight.com/ResourcesRest/Content/?isBinary=1&saveAs=Sharperlight+5.4.61+32bit+WINDOWS+32BIT+ONLY.msi&file=_hashKeywtDLbrZX8kShBPE%2bzumlKCOs6CMm381V%2fRkOd6Y0Mqk%2fdt%2bCJBdoNlVivTGdJfkXDEe8C0b1DZ3%2biS%2fxmdY%2bTqlJ%2b0Q8Rk5kJRSEjnZ8HgoL%2b9hFjGtKpsD7jUBGMqTdQjFCUFVk91U%3d
+  InstallerSha256: 39D5614F6796F1F3254DA4628EBACC8C9D954157E312011DAA9D140EA658A76F
```
```suggestion
  InstallerSha256: 2F1E4C61B4A47D56E2C6B27E59F665663D4C92901F42DEEC2F0DB16971005134

```

DeusMaximus committed:
Update manifests/p/PhilightPtyLtd/Sharperlight/5.4.60/PhilightPtyLtd.Sharperlight.installer.yaml

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

DeusMaximus committed:
Remove 32-bit installer.

The 32-bit installer uses a different version (5.4.61) and is otherwise painful.

As per the install guide found at https://www.sharperlight.com/builds-released/, the intention is to only install the 32-bit build on 32-bit machines only, and to always use 64-bit builds on 64-bit machines. If necessary, we could add the 32-bit build as a seperate ID (PhilightPtyLtd.Sharperlight32bit or whatever) later if someone needs it.
